apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -906,5 +906,9 @@ export async function updateLti13Scores(
   job.info('Done.\n\nSummary:');
   job.info(`${counts.success} score${counts.success === 1 ? '' : 's'} successfully sent.`);
   job.info(`${counts.error} score${counts.error === 1 ? '' : 's'} not sent due to errors.`);
+  if (counts.error > 0 && counts.success === 0) {
+    job.warn(`\tNo successful scores sent and errors are present.`);
+    job.warn(`\tIs the ${instance.lti13_instance.name} course published?`);
```

Comment:
"published" smells like Canvas-specific terminology, though I'm not sure what else to do here. I'm fine sticking with this for now.

Reply:
Yeah, this is something we might have to revisit when we add more LMS support and tips get more platform specific.